src/design-system/components/table/index.tsx

```diff
@@ -142,16 +153,40 @@ const Table = ({ virtualizedParams, ...props }: TableProps) => {
                 rowCount={props.rows.length}
                 rowHeight={virtualizedParams.rowHeightPx}
                 overscanRowCount={virtualizedParams.overscanRowCount}
-                rowRenderer={({ index, key, style }) => (
-                  <Box key={key} style={style}>
-                    <TableRow hover={!!props.rowHover} cells={props.rows[index] || []} />
-                  </Box>
-                )}
+                rowRenderer={({ index, key, style }) => {
+                  const row = props.rows[index] || [];
+                  if ("cells" in row) {
+                    return (
+                      <Box key={key} style={style}>
```

Comment:
I wonder if this should also use `row.key`. Did you check that the fix applies to the virtualized table too?
```suggestion
                      <Box key={row.key} style={style}>
```

Reply:
updated